File: android/src/main/java/com/valentingrigorean/arcgis_maps_flutter/measure/ArcgisMeasureHelper.kt
package com.valentingrigorean.arcgis_maps_flutter.measure

import com.valentingrigorean.arcgis_maps_flutter.workaround.FlutterWorkAround

interface ArcgisMeasureHelper :FlutterWorkAround{
    fun initMeasure()
    fun reset()
    fun clear(): Double
    fun setMeasureMode(mode: MeasureMode)
    fun revoke(): Double
    fun makePoint(): Double
}

File: ios/Classes/Measure/ArcGisMeasureHelper.swift
//
//  ArcGisMeasureHelper.swift
//  arcgis_maps_flutter
//
//  Created by Mo on 2022/7/18.
//
protocol ArcGisMeasureHelper {
    func initMeasure()
    func reset()
    func clear() -> Double
    func revoke() -> Double
    func makePoint() -> Double
}
